src/api/auth/_dto/auth.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { IsEmail, IsNotEmpty, MinLength, IsString } from 'class-validator';
+
+export class LoginUserDto {
+  @IsNotEmpty({ message: 'Field email cannot be empty' })
```

Comment:
you dont have specify message. we only need error messages for custom validation decorators